strings.txt

```diff
@@ -20991,7 +20991,7 @@ SETUP_DEBUG_SERVER_LOG_DESC
 	FI	Logitech Media Server pitää lokitiedostoa kaikista sovelluksiin liittyvistä toiminnoista (äänen virtautus, infrapuna jne.) täällä:
 	FR	Le Logitech Media Server conserve un fichier journal de toutes les activités liées à l'application (diffusion audio, infrarouge, etc.), à l'emplacement suivant :
 	IT	Logitech Media Server mantiene un file di registro per tutte le attività relative alle applicazioni (stream audio, raggi infrarossi e così via) nel seguente percorso:
-	NL	Logitech Media Server bewaart hier een logboek voor alle toepassingsactiviteiten (audio-streaming, infrarood, enz.):
+	NL	Logitech Media Server bewaart hier een logboek voor alle applicatie-activiteiten (audio-streaming, infrarood, enz.):
```

Comment:
applicatieactiviteiten is without a dash, please remove the dash.

Reply:
Thanks for the review. Do I need to add the fix to this PR?